infrastructure/demo.tfvars

```diff
@@ -11,4 +11,5 @@ data_store_max_pool_size = 48
 database_sku_name = "GP_Gen5_8"
 database_sku_capacity = "8"
 
-definition_cache_latest_version_ttl_sec = 30
\ No newline at end of file
+definition_cache_latest_version_ttl_sec = 30
```

Comment:
can we have 2 min on Demo?

Reply:
Done for both jurisdiction and last version caches on both prod and demo.